microservice/challenge-4/cat-mood-producer/src/main/java/ca/antaki/www/cat/producer/business/MoodGenerator.java

```diff
@@ -0,0 +1,15 @@
+package ca.antaki.www.cat.producer.business;
+
+import java.util.concurrent.ThreadLocalRandom;
+
+import org.springframework.stereotype.Component;
+
+import ca.antaki.www.cat.producer.model.CatMood;
+
+@Component
+public class MoodGenerator {
+
+	public byte generateMood() {
+		return (byte) ThreadLocalRandom.current().nextInt(CatMood.MIN, CatMood.MAX);
```

Comment:
can you explain the usage of `ThreadLocalRandom` ?

Reply:
I used ThreadLocalRandom for performance as this will be called from multiple threads and ThreadLocalRandom will have one Random per thread